tenant-isolation-config-service-impl/src/main/java/org/hypertrace/tenantisolation/config/service/store/TenantIsolationGroupConfigDocument.java

```diff
@@ -0,0 +1,38 @@
+package org.hypertrace.tenantisolation.config.service.store;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.hypertrace.core.documentstore.Document;
+
+public class TenantIsolationGroupConfigDocument implements Document {
+
+  private static final ObjectMapper OBJECT_MAPPER =
```

Comment:
if we had just read/written the proto, we wouldn't have to manually map this either.

Reply:
fixed by directly using proto <-> json